content/20.treatments.md

```diff
@@ -540,19 +540,59 @@ Although exciting, the cost of vaccine manufacturing and electroporation may mak
 
 #### RNA Vaccines
 
-Brief background on the therapeutic.
+RNA vaccines are nucleic-acid based modalities that code for viral antigens against which the human body elicits a humoral and cellular immune response. 
+The mRNA technology is transcribed _in vitro_ and delivered to cells via lipid nanoparticles (LNP). 
+They are recognized by ribosomes in vivo and then translated and modified into functional proteins [@doi:10.3389/fimmu.2019.00594]. 
+The resulting intracellular viral proteins are displayed on surface MHC I proteins, provoking a strong CD8+ T cell response as well as a CD4+ T cell and B cell-associated antibody responses [@doi:10.3389/fimmu.2019.00594]. 
```

Comment:
It's definitely correct that presentation on MHC I will generate a CD8 T cell response, but I think to get the CD4 T cell response (and subsequent Ab induction) you need signaling through the MHC II pathway as well. I suggest clarifying that here, or perhaps just saying "MHC proteins" instead of "MHC I proteins" here?